### PR TITLE
Add logging for all container/sandbox responses

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel mpatel@redhat.com

### CODE REVIEW

File: server/sandbox.go [hunk 1]
```diff
@@ -421,12 +425,13 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 	s.releasePodName(sb.name)
 	s.removeSandbox(sb.id)
 
-	return &pb.RemovePodSandboxResponse{}, nil
+	resp := &pb.RemovePodSandboxResponse{}
```
Comment (runcom) [inline server/sandbox.go]: Missing logrus call below

File: server/container.go [hunk 2]
```diff
@@ -293,7 +296,7 @@ func (s *Server) createSandboxContainer(containerID string, containerName string
 	// Join the namespace paths for the pod sandbox container.
 	podInfraState := s.runtime.ContainerStatus(sb.infraContainer)
 
-	logrus.Infof("pod container state %v", podInfraState)
+	logrus.Debugf("pod container state %v", podInfraState)
```
Comment (runcom) [inline server/container.go]: Missing `+` in `%v`

**Comment by runcom** [comment]
Small nits, LGTM otherwise

**Comment by rhatdan** [comment]
Other then @runcom comments LGTM.  Fix those and merge.

**Comment by mrunalp** [comment]
Updated.

**Comment by runcom** [comment]
CI seems stuck :confused:

**Comment by mikebrow** [comment]
LGTM

**Comment by mrunalp** [comment]
Restarted build.

**Comment by mrunalp** [comment]
@runcom PTAL

**Comment by runcom** [comment]
LGTM

**Pull request merged**